tests/test_dependencies.py

```diff
@@ -0,0 +1,47 @@
+"""Tests for Ralph flavors dependencies"""
+
+import importlib
+import re
+import sys
+
+import pytest
+from click.testing import CliRunner
+
+
+def test_dependencies_ralph_command_requires_click(monkeypatch):
+    """Tests Click module installation while executing the ralph command."""
+
+    monkeypatch.setitem(sys.modules, "click", None)
+
+    # Force ralph.cli reload now that click is considered as missing
+    if "ralph.cli" in sys.modules:
+        del sys.modules["ralph.cli"]
+
+    with pytest.raises(
+        ModuleNotFoundError,
+        match=re.escape(
+            "You need to install 'cli' optional dependencies to use the ralph "
+            "command: pip install ralph-malph[cli]"
+        ),
+    ):
+        importlib.import_module("ralph.cli")
```

Comment:
We could avoid the if condition by importing `from ralph import cli` and changing

```suggestion
    with pytest.raises(
        ModuleNotFoundError,
        match=re.escape(
            "You need to install 'cli' optional dependencies to use the ralph "
            "command: pip install ralph-malph[cli]"
        ),
    ):
        importlib.reload(cli)
```
WDYT?

Reply:
At the end, your suggestion produces the same effect, but I found it less explicit.